content/en/tracing/profiling/_index.md

```diff
@@ -138,6 +201,20 @@ Once enabled, the following profile types are collected:
 | Socket I/O               | Shows the time each method spent handling socket I/O.                                                                                                                                                                                                                                              |
 
 
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+Once enabled, the following profile types are collected:
+
+| Profile type             | Definition                                                                                                                                                                                                                                                                                         |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU         | Shows the time each method spent running on the CPU. It includes CPython bytecode, but not native code called from within Python.                                                                                                                                                                     |
```

Comment:
Actually it does include native code. Did we write that somewhere else?
TBH I don't think we should use the word "native". It might refer to something in Java, but I never heard of it in the Python ecosystem.

Reply:
Corrected

Reply:
Awesome! I was a bit surprised to see this :-)